platform/security/filter/security-filter-web-sso/src/main/java/org/codice/ddf/security/filter/websso/WebSSOFilter.java

```diff
@@ -138,10 +149,16 @@ private void handleRequest(HttpServletRequest httpRequest, HttpServletResponse h
             FilterChain filterChain, List<AuthenticationHandler> handlers)
             throws IOException, ServletException {
 
-        // First pass, see if anyone can come up with proper security token from
-        // the git-go
+        if (handlers.size() == 0) {
+            LOGGER.warn("Handlers not ready. Returning status code 503, Service Unavailable.");
```

Comment:
org.codice.ddf.security.filter.websso.WebSSOFilterTest.testDoFilterForbidden fails here as it is still expecting a 400 code.